ArduCopter/autoyaw.cpp

```diff
@@ -323,17 +323,17 @@ float Mode::AutoYaw::rate_cds()
     }
 
     // return zero turn rate (this should never happen)
-    return _yaw_rate_cds;
+    return _yaw_rate_rads;
 }
 
 AC_AttitudeControl::HeadingCommand Mode::AutoYaw::get_heading()
```

Comment:
Maybe add units to this function name as well? (e,g, `get_heading_rad()`.  I appreciate that `HeadingCommand` has angle and rate in it, but reading this function when used in the code I would assume that "heading" would be in degrees as headings are very commonly quoted in degrees in manned navigation applications (e.g. sailing, hiking, etc).  Certainly not a blocker, just offering a perspective is all.